### PR TITLE
Correct path to public Simics simulator + more

### PR DESCRIPTION
**Pull request by jakob-engblom-i**
Update readme to point at the correct location for the public release of the Intel Simics simulator.  Added a bit more substance to the introduction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,34 +5,60 @@
...
 
 ### DMLC_PATHSUBST
 The DMLC build copies a few DML library files, e.g. `dml-builtins.dml`, into
-`linux64/bin`. When a compile error happens, error messages will normally point
+`<hosttype>/bin`. When a compile error happens, error messages will normally point
 to this copy rather than the source. By setting `DMLC_PATHSUBST` to
-`linux64/bin/dml=modules/dmlc/lib`, error messages will be rewritten to point
-to the source file instead.
+`<hosttype>/bin/dml=modules/dmlc/lib`, error messages will be rewritten to point
+to the source file instead.  `<hosttype>` is either `linux64` or `win64` depending on your host type.
```
Comment (lwaern-intel) [inline README.md:61]: Paragraph exceeds 80 character width.

File: README.md [hunk 2]
```diff
@@ -5,34 +5,60 @@
...
-## Building and testing
+Currently, the compiler supports building models for the Intel®
+[Simics®](https://www.intel.com/content/www/us/en/developer/articles/tool/simics-simulator.html)
+simulator, but other back-ends may be added in the
+future.
+
+
+## Building and testing DMLC 
+
+To build the DML compiler (DMLC), you need to have a Simics
```
Comment (mandolaerik) [inline README.md:24]: You already said (DMLC) in the intro section, do we need to repeat it?

File: README.md [hunk 3]
```diff
@@ -5,34 +5,60 @@
...
+simulator installation and a Simics project set up.
+
+
+### Using the Public Release of the Intel Simics Simulator
+
+If you do not already have a Simics simulator installation or access to the
+Simics simulator via commercial channels, install the [Public Release of
+the Intel Simics simulator](https://software.intel.com/simics-simulator) and create a Simics project (automatic in the default installation flow).
```
Comment (mandolaerik) [inline README.md:32]: Should we sprinkle some ®:s here?

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9780815

File: README.md (hunk 3 above)
Comment (jakob-engblom-i) [reply README.md:32]: No, only needed on first mention in a text.

File: README.md (hunk 2 above)
Comment (jakob-engblom-i) [reply README.md:24]: Not really.

**Review by lwaern-intel — APPROVED** [review]

**Review by mandolaerik — APPROVED** [review]

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9782712

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9782711

**Pull request merged**